Dockerfile.task_base

```diff
@@ -9,31 +9,6 @@ ENV PIP_NO_CACHE_DIR=1
 
 RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
 
-RUN apt-get update && apt-get install -y software-properties-common
-
-RUN add-apt-repository ppa:ubuntugis/ppa && \
```

Comment:
I don't know what this was for, but it's apparently not necessary for installing pypgstac.

I also removed curl and git. We no longer have any dependencies that require installing from `git`. If / when we do, we might have other ways to get it into the container (including building a wheel on the host).